features/lwipstack/LWIPInterface.cpp

```diff
@@ -429,7 +467,7 @@ nsapi_error_t LWIP::add_l3ip_interface(L3IP &l3ip, bool default_if, OnboardNetwo
 #if LWIP_IPV4
                    0, 0, 0,
 #endif
-                   interface, &LWIP::Interface::emac_if_init, ip_input)) {
+                   interface, &LWIP::Interface::l3ip_if_init, ip_input)) {
```

Comment:
This is problematic for non-ethernet interfaces (for example cellular). `LWIP::Interface::l3ip_if_init` sets interface as ETHERNET: 
```
    /* Interface capabilities */
    netif->flags = NETIF_FLAG_BROADCAST | NETIF_FLAG_ETHARP | NETIF_FLAG_ETHERNET;
```

Reply:
I think l3ip_if_init should only set NETIF_FLAG_BROADCAST flag.

Reply:
Right. Fixed now